WoWPro_Leveling/Classic/Horde/21_30_Jame_Horde.lua

```diff
@@ -145,7 +145,7 @@ A Elixir of Suffering |QID|499|PRE|496|M|61.40,19.10|Z|Hillsbrad Foothills|N|Fro
 T Elixir of Suffering |QID|499|M|61.40,19.10|Z|Hillsbrad Foothills|N|At Umpi.|
 
 C Elixir of Pain|QID|501|S|N|Kill every Starving Mountain Lion you find.|
-R Hillsbrad Fields|QID|527|M|32.20,41.30|Z|Hillsbrad Foothills|
+R Hillsbrad Fields|ACTIVE@|527|M|32.20,41.30|Z|Hillsbrad Foothills|
```

Comment:
Typo?

Reply:
thats what happens when you have 10 windows open, i wondered where that @ sign went.